components/style/properties.mako.rs

```diff
@@ -2461,7 +2469,8 @@ pub mod longhands {
     </%self:single_keyword_computed>
 
     // TODO(pcwalton): `full-width`
-    ${single_keyword("text-transform", "none capitalize uppercase lowercase")}
+    ${single_keyword("text-transform", "none capitalize uppercase lowercase",
+                     extra_gecko_values="full-width")}
```

Comment:
Here's something that's really, _really_, facepalm-worthy - the [Gecko constant property](http://searchfox.org/mozilla-central/source/layout/style/nsCSSProps.cpp#2047) deviated from convention and removed the underscore between FULL and WIDTH. I'm not sure how to handle this case here.